app/Models/RequestCanvassSummary.php

```
@@ -82,39 +83,27 @@ public function getGrandTotalAmountAttribute(): float
         $quotations = $procurement->priceQuotations()
             ->with([
                 'supplier',
-                'items' => fn ($q) => $q->orderBy('id')
+                'items' => fn ($q) => $q->whereNotNull('unit_price')->orderBy('id')
             ])
             ->latest()
             ->take(3)
             ->get();
-        // Load requisition slip items and index by item_id
         $procurement->loadMissing('requisitionSlip.items.itemProfile');
         $reqItems = $procurement->requisitionSlip->items->keyBy('item_id');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$reqItems'. (undefined)

(UnusedLocalVariable)